broker/providers/ansible_tower.py

```diff
@@ -366,36 +372,42 @@ def _compile_host_info(self, host):
             "tower_inventory": self._translate_inventory(host.inventory),
             "_broker_provider": "AnsibleTower",
             "_broker_provider_instance": self.instance,
-            "_broker_args": getattr(host, "_broker_args", {}),
+            # Get _broker_args from host facts if present
+            "_broker_args": getattr(host_facts, "_broker_args", {}) or self._get_broker_args_from_job(host),
```

Comment:
since this one is just the single fallback, can just have it be the default for get.
```suggestion
            "_broker_args": getattr(host_facts, "_broker_args", self._get_broker_args_from_job(host)),
```